api.ts

```diff
@@ -12012,16 +12012,12 @@ export const DefaultApiAxiosParamCreator = function (configuration?: Configurati
         /**
          * You can upload ID photos to this endpoint. Like document upload, files must be uploaded as a multipart form. Valid file types are jpg, png and pdf. The file size must be between 32KB and 10MB. 
          * @summary Upload ID photo
-         * @param {string} applicantId The ID of the applicant whose ID photo is being uploaded.
-         * @param {File} file The file to be uploaded.
+         * @param {string} [applicantId] The ID of the applicant whose ID photo is being uploaded.
+         * @param {File} [file] The file to be uploaded.
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        uploadIdPhoto: async (applicantId: string, file: File, options: RawAxiosRequestConfig = {}): Promise<RequestArgs> => {
-            // verify required parameter 'applicantId' is not null or undefined
-            assertParamExists('uploadIdPhoto', 'applicantId', applicantId)
-            // verify required parameter 'file' is not null or undefined
-            assertParamExists('uploadIdPhoto', 'file', file)
+        uploadIdPhoto: async (applicantId?: string, file?: File, options: RawAxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
This change (and next ones) will need some investigation, since these fields are still [required](https://github.com/onfido/onfido-openapi-spec/blob/master/schemas/id_photos/id_photo_uploader.yaml#L1-L3).